src/lib/server.ts

```diff
@@ -78,6 +83,8 @@ export const buildServer: (options: {
   server.register(fastifyBlipp);
   server.register(fastifyRateLimit, {
     allowList: ["127.0.0.1"],
+    cache: 5000,
+    redis,
```

Comment:
Check if cache makes sense with redis.

Reply:
Thanks. should have no effect on redis, since it is a setting related to lru in memory cache.